pkg/storage/fs.go

```diff
@@ -156,6 +169,37 @@ func (fsc *fileSystemBackend) SavePrivateKey(keyIdentifier types.KeyIdentifier,
 	return err
 }
 
+// GetExpiringCertificates naive implementation, uses the fileTree to check every certificate
+// it'll not group certificates based on legalEntity. This is up to a more robust implementation
+func (fsc *fileSystemBackend) GetExpiringCertificates(from time.Time, till time.Time) ([]*x509.Certificate, error) {
+	if from.Equal(till) || from.After(till) {
+		return nil, ErrInvalidDuration
+	}
+
+	var expiringCertificates = make([]*x509.Certificate, 0)
+
+	// list all files ending on certificateEntry in fcs.path
+	err := filepath.Walk(fsc.fspath, func(path string, info os.FileInfo, err error) error {
+		// only target expiringCertificates
+		if strings.Contains(info.Name(), string(certificateEntry)) {
+			x509, err := fsc.readCertificate(path)
```

Comment:
x509 is a package name which is discouraged to use as variable name. Rename to cert/certificate?

Reply:
cert is also a package, will use certificate